packages/sdk/src/modules/storage/directory.ts

```diff
@@ -54,7 +54,7 @@ export class Directory extends ApillonModel {
   ) {
     super(directoryUuid);
     this.bucketUuid = bucketUuid;
-    this.API_PREFIX = `/storage/${bucketUuid}`;
+    this.API_PREFIX = `/storage/buckets/${bucketUuid}/directories/${directoryUuid}`;
```

Comment:
Why is this not: `this.API_PREFIX = /storage/buckets/${bucketUuid}` Then the prefix used in get? I know that base here is directory but if you need a lower level prefix for an api call it should probably stay as that lower level.